src/components/Home/KeyFeatures.jsx

```diff
@@ -22,28 +22,19 @@ const KeyFeatures = ({ icon, title, description, isReallySmallScreen }) => (
       sx={{
         display: 'flex',
         alignItems: 'center',
+        gap: '10px',
         marginBottom: '14px',
-        color: 'primary.main',
-        width: isReallySmallScreen ? 1 : 'full'
+        color: 'primary.main'
       }}
     >
       {icon}
-      <Typography
-        variant={isReallySmallScreen ? 'h5' : 'h4'}
-        sx={{
-          marginLeft: '8px',
-          textAlign: isReallySmallScreen ? 'start' : 'center',
-          width: '100%'
-        }}
-      >
-        <strong>{title}</strong>
-      </Typography>
+      <strong>{title}</strong>
```

Comment:
I think this is a good case where simpler is better. `<strong>` is relatively straightforward and does the intended effect without relying on MUI and extra lines of code.

Reply:
<strong> is also better for screenreaders and other accessibility programs than font weight. That's actually why the w3c named the element "strong" instead of "bold"